documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -11,46 +11,221 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, d
+        '{' <language-tag>':' <value-string> '}'
+   ']'
+
+<value-only-of-Range> ::=  
+    '{  
+		"min":' <value of Range>',' 
+       '"max":' <value of Range>
+    '}'
+
+<value-of-ReferenceElement> ::= 
+    <value-Reference>
+
+<value-of-File> ::= 
+    '{
+        "Contenttype":' <value-ContentType>','
+       '"value":' <value-PathType>
+    '}'
+
+<value-of-Blob> ::= 
+    '{'
+       '"ContentType":' <value-ContentType>[','
+       '"value":' <value-BlobType> ]
+    '}'
+
+<value-only-of-RelationshipElement> ::= 
+    '{
+        "first":' <value-Reference>','
+       '"second":' <value-Reference> 
+    '}'
+    
+<value-only-of-AnnotatedRelationshipElement> ::=
+    '{
+        "first"': <value-Reference>','
+       '"second"': <value-Reference>[','
+       '"annotations": ['  
+            {<jd><idShort-of-sme><jd>':' <value-only-of-DataElement>','}*  
+             <jd><idShort-of-DataElement><jd>':'  <value-only-of-sme>
+        ']'
+    '}'
+        
+<value-only-of-Entity> ::=
+    '{'
+      ['"statements": ['
+          {<jd><idShort-of-sme><jd>':' <value-only-of-sme>','}*  
+           <jd><idShort-of-sme><jd>':' <value-only-of-sme>
+        '],']
+       '"entityType":' <value-EntityType>[',' (
+       '"globalAssetId":' <value-Identifier>[','
+       '"specificAssetIds":' <value-of-specificAssetIds>','] | 
+       '"specificAssetIds":' <value-of-specificAssetIds> )
+       ]
+    '}'
+    
+<value-only-of-BasicEvent> ::= 
+    '{
+        "observed":' <value-Reference>
+    '}'
+    
+<value-of-Property> ::= 
+	value of attribute Property/value with data type as specified in Property/valueType. 
+	In case of data type "xs:string" delimiters '"' are added (e.g. "this-is-mystring")
+
+<value-of-Range> ::= 
+	value of Range/min or Range/max, resp. 
+	with data type as specified in Range/valueType
+
+<value-Reference> ::= 
+    '{'
+       '"type":' <value-of-Reference-type>','
+       '"keys":' <value-of-Reference-keys>
+    '}'
+
+<idShort-of-sme> ::= 
+	value of SubmodelElement/idShort
+
+<language-tag> ::= 
+	language tag as defined for values of type langString, i.e.
+	in accordance with IETF https://tools.ietf.org/html/rfc5646#page-5[RFC 5646], the language names match the following regular expression:
+
+		^[a-z]\{2,4}(-[A-Z][a-z]\{3})?(-([A-Z]\{2}|[0-9]\{3}))?$
+	
+	e.g. "en" for Englisch
+
+<value-of-Reference-type> ::= 
+	value of Reference/type
+
+<value-of-Reference-keys> ::= 
+	"Normal" JSON serialization of list of elements of type Key
+
+<value-of-specificAssetIds> ::= 
+	"Normal" JSON serialization of list of elements of type SpecificAssetId
+
+<value-string> ::= 
+	value of type string with delimiters '"', e.g. "this is my string"
+
+<value-PathType> ::= 
+	value of type PathType
+
+<value-BlobType> ::= 
+	value of type BlobType
+
+<value-Identifier> ::= 
+	value of type Identifier
+
+<jd> ::= 
+	<json-delimiter>  
+
+<sd> ::= 
+	<string-delimiter>  
+
+<json-delimiter> ::= 
```

Comment:
Conflate with `<jd>` definition - only have one of both and do not "chain" them together. This just complicates things more than necessary. Maybe even remove them completely and use literals `'"'` instead

Reply:
removed